src/App.css

```diff
@@ -36,3 +36,174 @@
+
+
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300;700&family=Open+Sans:wght@300;400;500;600;700;800&family=Raleway:ital,wght@0,200;0,400;0,600;0,700;0,800;0,900;1,500&family=Roboto+Mono:wght@300;400;500;600;700&family=Roboto+Serif:ital,opsz,wght@0,8..144,100;0,8..144,300;1,8..144,100;1,8..144,200&display=swap');
+.App {
+  text-align: center;
+  height: 100vh;
+  width: 100vw;
+  padding: 0;
+  margin: 0;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center; 
+  /* justify content centre to make the box centre vertically in y axis--> centre of the column  */
+  align-items: center; 
+  /* align items centre to make the box centre in x axis--> centre of the row  */
+  font-family: "Poppins", sans-serif;
+  color: var(--primary-color);
+}
+
+main{
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.3);
+  /* font-family: 'Merriweather', serif;
+font-family: 'Open Sans', sans-serif;
+font-family: 'Raleway', sans-serif;
+font-family: 'Roboto Mono', monospace;
+font-family: 'Roboto Serif', serif; */
+}
+.main-bar{
+  padding: 2rem;
+  align-content: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-weight:bold;
+}
+.onboarding-head{
+  font-size: 5rem;
+}
+.onboarding-input{
+  font-size: 2.5rem;
+  padding: 1rem;
+  color: white;
+  background-color: transparent;
+  border:0;
+  border-bottom: 2px solid var(--primary-color);
+  /* font-size: large; */
+  text-align: center;
+}
+.onboarding-input::placeholder{
+  color: white;
+}
+.onboarding-input:focus{
+  outline: none;
+}
+.onboarding-input:focus::placeholder{
+  color: transparent;
+}
+.footer-bar{
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  top: 5rem;
+  padding: 2rem;
+  justify-content: center;
+}
+.greeting{
+  font-size: 3rem;
+  margin:2rem;
+}
+
+.quote{
+  padding: 1rem;
+  font-size: 3rem;
+  font-weight: 300;
+  background-color: rgba(0, 0, 0, 0.3);
+  justify-content: center;
+}
+.author{
+  text-align: end;
+  padding: 1rem;
+  font-size: 2rem;
+  font-weight: 400;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+.clock p{
+  font-size: 8rem;
+}
+
+.focus-input{
+  padding: 1rem;
+  color: white;
+  background-color: transparent;
+  border:0;
+  border-bottom: 2px solid var(--primary-color);
+  font-size: large;
+  font-size: 2.5rem;
+  text-align: center;
+}
+.focus-input::placeholder{
+  color: white;
+}
+.focus-input:focus{
+  outline: none;
+}
+.focus-input:focus::placeholder{
+  color: transparent;
+}
+.floating-btn{
+  border: 0;
+  background-color: transparent;
+}
+.floating-btn:hover{
+  background-color: rgba(247, 241, 241, 0.5);
+  /* color:black; */
```

Comment:
remove the commented code

Reply:
> left few comments please look into it. Other than that good to merge. UI looks good.

Thanks @ianuragsharma for reviewing. I made the necessary changes pointed out in file